typescript/src/workflows/AccountTransferWorkflow/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  proxyActivities,
+  sleep,
+  workflowInfo,
+  defineQuery,
+  setHandler,
+  log,
+  uuid4,
+  RetryPolicy,
+} from '@temporalio/workflow';
+import type * as activities from '../../activities/index';
+import type { DepositResponse, TransferInput, TransferOutput, TransferStatus } from "../../types";
+
+const DEFAULT_RETRY_POLICY: RetryPolicy = {
+  initialInterval: '1s',
+  maximumInterval: '30s',
+  backoffCoefficient: 2,
+};
+
+const { validateAsync, withdrawAsync, depositAsync, sendNotificationAsync } =
+proxyActivities<typeof activities>({
+  startToCloseTimeout: '5s',
+  retry: DEFAULT_RETRY_POLICY,
+});
+
+const transferStatusQuery = defineQuery<TransferStatus>('transferStatus');
+
+export async function AccountTransferWorkflow(input: TransferInput): Promise<TransferOutput> {
+  const { workflowType } = workflowInfo();
+
+  let progress:number, transferState:string;
+  let depositResponse:DepositResponse = {
+    chargeId: ''
+  }
+
+  setHandler(transferStatusQuery, () => {
+    return {
+      transferState,
+      progressPercentage: progress,
+      workflowStatus: '',
+      chargeResult: depositResponse,
+      approvalTime: 0
```

Comment:
This should be either a variable or set to 30, otherwise the UI won't show the progress.

Reply:
But, since this is not the scenarios workflow, this value is never used. Might be better to put a comment here to indicate that